README.md

```diff
@@ -11,23 +11,39 @@ It is a very small version of a full Snowplow data collection pipeline: small en
 
 ## Quick start
 
-Snowplow Micro is configured at runtime through the [Stream Collector][stream-collector] configuration and the [Iglu][iglu] resolver configuration. So, to start Snowplow Micro:
-
-1. Update the [stream collector configuration][stream-collector-config] for Snowplow Micro: [example configuration][collector-config-example].
-2. Update the [Iglu resolver configuration][iglu-resolver-config] for Snowplow Micro: [example configuration][iglu-resolver-example].
-3. Run Micro either using Docker or using Java.
-
 ### Using Docker
 
 Snowplow Micro is hosted on Docker Hub ([snowplow/snowplow-micro][docker-micro]) with images available for both `amd64` and `arm64` architectures.
 
+The following command can be used to run Micro with the default configuration:
+
+```bash
+docker run -p 9090:9090 snowplow/snowplow-micro
+```
+
+A [distroless][distroless-repo] image is also provided, which benefits from being smaller and more secure, with the downside of basic utilities (such as a shell) not being present in the image.
+
+The distroless image is tagged as `snowplow/snowplow-micro:x.x.x-distroless` on Docker Hub.
```

Comment:
Sooner or later a user will tell us they can't pull the `x.x.x-distroless` image.  Seriously, that will happen.

I suggest replace it with `1.3.0-distroless`.